tools/istio-docker.mk

```diff
@@ -66,14 +66,18 @@ else
 	cp ${ISTIO_ENVOY_LINUX_RELEASE_PATH} ${ISTIO_ENVOY_LINUX_RELEASE_DIR}/envoy
 endif
 
+# The file must be named 'envoy_bootstrap_v2.json' because Dockerfile.proxyv2 hard-codes this.
+${ISTIO_ENVOY_BOOTSTRAP_CONFIG_DIR}/envoy_bootstrap_v2.json: ${ISTIO_ENVOY_BOOTSTRAP_CONFIG_PATH}
+	cp ${ISTIO_ENVOY_BOOTSTRAP_CONFIG_PATH} ${ISTIO_ENVOY_BOOTSTRAP_CONFIG_DIR}/envoy_bootstrap_v2.json
```

Comment:
In the code `ISTIO_ENVOY_BOOTSTRAP_CONFIG_DIR` is being set to `dir ISTIO_ENVOY_BOOTSTRAP_CONFIG_PATH`. So isn't this always going to be `cp X X` and not actually do anything?

Reply:
if the name of the file provided by the user is not envoy_bootstrap_v2.json, this will make a copy of the file in the same dir and name it envoy_bootstrap_v2.json

Reply:
I see. thanks1